blog/content/post/2024-03-15-uefi-sb-implementation-mpl.md

```diff
@@ -0,0 +1,385 @@
+---
+title: Implementing UEFI Secure Boot on MPL PIP4x
+abstract: "This post explains how we tackled the problem of implementing UEFI
+           Secure Boot on MPL's PIP platforms. The process included testing the
+           platform's compatibility with Secure Boot and integrating automatic
+           image signing into an existing Yocto layer."
+cover: /covers/mpl-pip44.jpg
+author: pawel.langowski
+layout: post
+published: true
+date: 2024-03-15
+archives: "2024"
+
+tags:
+  - secure boot
+  - uefi
+  - mpl
+  - yocto
+categories:
+  - Firmware
+  - Security
+
+---
+
+MPL is a Swiss company, which designs and manufactures embedded computers and
+microcontroller hardware for rugged environment, extended temperature range,
+and with long-term availability. The PIP series is a family of low-power,
+read-to-use embedded computers manufactured by MPL. Recently we tackled the
```

Comment:
```suggestion
ready-to-use embedded computers manufactured by MPL. Recently we tackled the
```

@PLangowski please run grammarly on that blog post to get any typos

Reply:
16f72d0ee96a1a1d242206f9583ebf32126162f6